builtin/providers/azurerm/resource_arm_virtual_machine.go

```diff
@@ -190,7 +208,24 @@ func resourceArmVirtualMachine() *schema.Resource {
 
 						"vhd_uri": {
 							Type:     schema.TypeString,
-							Required: true,
+							Optional: true,
```

Comment:
Hey @brandontosch

> I submitted the ConflictsWith fix under PR #13019 and removed it from this branch.

Great, thanks!

> Do you think it will be alright to leave the ConflictsWith setup in place here with the hope that the other PR would get merged in time for the release of this feature or should I revert this back to the custom conflict enforcement logic? Or would it be alright if the conflict enforcement doesn't work until later? Currently, if you use both fields, you'll get a bad request error from the azure api when terraform attempts to create the resource.

Given this PR otherwise looks good to merge - rather than merge this with no validation, could we update this to use the custom conflict logic for the moment? We can switch this over to using `ConflictsWith` function once the other PR is merged :)

Thanks!